js/analyzer.js

```diff
@@ -58,7 +58,7 @@ var Analyzer = function( args ) {
  */
 Analyzer.prototype.init = function( args ) {
 	if ( isUndefined( args.paper ) ) {
-		args.paper = new Paper( args.text, { keyword:  args.keyword, description: args.meta } );
+		args.paper = new Paper( args.text, { keyword:  args.keyword, description: args.meta, url: args.url, title: args.pageTitle } );
```

Comment:
For me it would be nice and more readable if the properties starts on a new line. Like:
```
args.paper = new Paper( 
    args.text, 
    { 
        keyword:  args.keyword, 
        description: args.meta, 
        url: args.url, 
        title: args.pageTitle 
    }
);```